frontend/test/test.helpers.ts

```diff
@@ -40,8 +43,53 @@ export const spyOnAxiosApiCallWithFile = (api, method) => {
 	});
 };
 
-export const createTestStore = () => createStore(combineReducers(reducers));
+export const createTestStore = () => {
+	let middlewares = undefined;
+	let waitingActions: Action[] | string[] = [];
+	let resolvePromiseObj = { resolvePromise: null };
+
+	const sagaMiddleware = createSagaMiddleware();
+	
+	middlewares = applyMiddleware(sagaMiddleware);
+
+	const discountMatchingActions =  (action: Action) => {
+		const waitingAction = waitingActions[0];
+	
+		if (action.type === waitingAction || isEqual(waitingAction, action)) {
```

Comment:
It's just an idea, but this could be re-written as 
```ts
if (_.isMatch(action, waitingAction)) { ... }
```
This way, line 139 of  `viewer.sagas.spec.ts` can be re-written as 
```ts
}, [DialogsActions.open('alert')]); // instead of [DialogsTypes.OPEN]
```
and be more consistent as we only pass actionCreators as waitingActions

Reply:
No, because dialog actions have more properties, like the props and I think is really annoying to have to match the exact error in here.